bool dummy
---
bool plan_executed
